### PR TITLE
[NUI] Apply Vector4 corner radius internally.

### PR DESCRIPTION
**Pull request by rabbitfor**
Signed-off-by: Jiyun Yang <ji.yang@samsung.com>

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/internal/Common/PropertyHelper.cs [hunk 1]
```diff
@@ -109,15 +110,14 @@ private static SearchResult GenerateVisualPropertySearchResult(View view, Visual
...
             {
                 property.Dispose();
-                return null;
+                return data.RelatedData == null ? null : GenerateVisualPropertySearchResult(view, data.RelatedData);
             }
 
             SearchResult result = new SearchResult(property, data.ObjectConverter, data.PropertyValueConverter);
 
-            while (data.RelatedData != null)
+            if (data.RelatedData != null)
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/Common/PropertyHelper.cs:118]: this looks to change the behavior of the `while() ... data = data.RelatedData` 
is this OK?

Comment (rabbitfor) [reply src/Tizen.NUI/src/internal/Common/PropertyHelper.cs:118]: Yes :) The previous code has a bug. It need to be fixed to remove `while`.

Comment (dongsug-song) [reply src/Tizen.NUI/src/internal/Common/PropertyHelper.cs:118]: OK, I got it~

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**